deploy/crds/enterprise.splunk.com_licensemasters_crd.yaml

```diff
@@ -694,6 +694,20 @@ spec:
                       same time
                     format: int64
                     type: integer
+                  phaseMaxTries:
+                    default: 3
+                    description: Max. number of tries used by the scheduler for a
```

Comment:
No, we haven't specified the details about the pipeline. Idea is to provide the default value and suggest this only when there is a real need to tune the config.

Reply:
This can be cryptic for someone looking to tweak. Either expose the config with clear explanation, or call it for internal/support purposes only or just don't expose it at all